Explorer/Assets/DCL/FeatureFlags/FeatureFlagsConfiguration.cs

```diff
@@ -5,8 +5,36 @@
 
 namespace DCL.FeatureFlags
 {
+    public enum FeatureFlags
+    {
+        PORTABLE_EXPERIENCE,
+        GLOBAL_PORTABLE_EXPERIENCE,
+        PORTABLE_EXPERIENCE_CHAT_COMMANDS,
+        MAP_PINS,
+        CUSTOM_MAP_PINS_ICONS
+    }
+
     public class FeatureFlagsConfiguration
     {
+        public static string GetFlag(FeatureFlags flags)
+        {
+            switch (flags)
+            {
+                case FeatureFlags.PORTABLE_EXPERIENCE:
+                    return "alfa-portable-experiences";
```

Comment:
Instead of using an enum, could you use string constants directly? So we avoid this conversion